features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/TimelinePresenter.kt

```diff
@@ -179,7 +180,14 @@ class TimelinePresenter @AssistedInject constructor(
                     resolveVerifiedUserSendFailureState.eventSink(ResolveVerifiedUserSendFailureEvents.ComputeForMessage(event.event))
                 }
                 is TimelineEvents.NavigateToRoom -> {
-                    navigator.onNavigateToRoom(event.roomId)
+                    // Workaround for not having the server names available, get possible server names from the user ids of the room members
```

Comment:
I was confused because I thought that `TimelineEvents.NavigateToRoom` was use to navigate to any roomId, but it is actually only used to navigate to the predecessor and to the sucessor room. Maybe rename to `TimelineEvents.NavigateToPredecessorOrSuccessorRoom`? (I cannot think of a common name to reflect the idea of Predecessor and Successor)

Reply:
Maybe `NavigateToRelatedRoom` with some doc comment? Or I can use the `NavigateToPredecessorOrSuccessorRoom` suggestion.

Reply:
Maybe `NavigateToLinkedRoom`?